.travis.yml

```diff
@@ -1,13 +1,23 @@
 language: node_js
 node_js:
-- '6'
-- '5'
-- '4'
-- '4.2.0'
```

Comment:
It may be worth updating https://github.com/thelounge/lounge/blame/master/README.md#L40 then.